subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/BuildVerdictPlugin.kt

```diff
@@ -0,0 +1,36 @@
+package com.avito.android.build_verdict
+
+import com.avito.kotlin.dsl.isRoot
+import com.avito.utils.logging.ciLogger
+import com.google.gson.GsonBuilder
+import org.gradle.api.Plugin
+import org.gradle.api.internal.project.ProjectInternal
+import org.gradle.kotlin.dsl.create
+
+internal data class TaskPath(val value: String)
```

Comment:
i believe gradle has a built-in public class for that

Reply:
Please, comment on decisions for resolved comments.
In this example, It's not clear why `org.gradle.util.Path` is not suitable.